src/components/application_manager/src/commands/mobile/subscribe_button_request.cc

```diff
@@ -58,7 +58,7 @@ void SubscribeButtonRequest::Run() {
 
   const mobile_apis::ButtonName::eType btn_id =
```

Comment:
@okozlovlux const name as follows should be kBtnId it think? If so, please check it everywhere.